client/ayon_core/hosts/substancepainter/api/lib.py

```diff
@@ -640,3 +642,88 @@ def _setup_prompt():
         return
 
     return project_mesh
+
+
+def get_export_presets_by_filtering(export_preset_name, channel_type_list):
```

Comment:
Also, refrain from marking the variable name as `channel_type_list` - you're better then off naming it `channel_types` or `channel_type_names` and providing type hints like `list[str]`